src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -873,4 +873,35 @@
      *        boolean value for 'useFmtOnly'.
      */
     void setUseFmtOnly(boolean useFmtOnly);
+
+    /**
+     * Returns the enclave attestation url used with enclave based Always Encrypted.
+     * 
+     * @return Enclave attestation url.
+     */
+    String getEnclaveAttestationUrl();
+
+    /**
+     * Sets the enclave attestation url to be used with enclave based Always Encrypted.
+     * 
+     * @param url
+     *        Enclave attestation url.
+     */
+    void setEnclaveAttestationUrl(String url);
+
+    /**
+     * Returns the enclave attestation protocol used with enclave based Always Encrypted.
```

Comment:
The official name of the feature is "Always Encrypted with Secure Enclaves". I have not encountered the term "enclave based Always Encrypted" anywhere, please update all instances.